jedi/evaluate/compiled/__init__.py

```diff
@@ -339,7 +339,11 @@ def dotted_from_fs_path(fs_path, sys_path):
     for s in sys_path:
         if (fs_path.startswith(s) and len(path) < len(s)):
             path = s
-    return _path_re.sub('', fs_path[len(path):].lstrip(os.path.sep)).replace(os.path.sep, '.')
+
+    # - Window
+    # X:\path\to\lib-dynload/datetime.pyd => datetime
+    module_path = fs_path[len(path):].lstrip(os.path.sep).lstrip('/')
```

Comment:
I think it should be:

    return _path_re.sub('', module_path).replace(os.path.sep, '.').replace('/', '.')

Because what if there are multiple `slashes, like `X:\path\to\something/foo/bar.py`?